### ISSUE TITLE
Software Request TensorFlow 1.8 (Nvidia container 18.07-py2)

### ISSUE THREAD

**Issue by stefanwayon** [jade-hpc-gpu/jade-hpc-gpu.github.io#68]
# Software Request

Hello,

Would it be possible to get this container installed? TensorFlow 1.8 adds some multi-GPU APIs that our code currently uses.

Thanks!
Ștefan

**Comment by LiamATOS** [comment]
py2 and py3 versions now installed

Thanks

Liam

**Comment by SliMM** [comment]
Awesome, thanks!

**Comment by botev** [comment]
I'm using the `small-partition` nodes, not sure however on which node this was run. Is there a specific command to choose the nodes correctly?

**Comment by SliMM** [comment]
yeah you can say `--exclude=dgj407` as an argument to sbatch, srun or in your launch script
